CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * Change `ax.plot` usage to `ax.scatter` in `plot_pair` ([1990](https://github.com/arviz-devs/arviz/pull/1990))
 
 ### Deprecation
+* Deprecated `fill_last`, `contour` and `plot_kwargs` arguments from `plot_pair` function ([2085](https://github.com/arviz-devs/arviz/pull/2085))
```

Comment:
I would use Removed here instead of deprecated (which I think is what we did when the warnings were added and the arguments were not used anymore)